app/javascript/packs/SearchMovie.jsx

```diff
@@ -32,33 +87,39 @@ export default class SearchMovie extends React.Component {
 
   render() {
     const movies = this.state.results;
-    const posterBasePath = "https://image.tmdb.org/t/p/w154"
+    const { store } = this.context;
+    const user = store.getState().user[0];
+
     return (
       <div className="container-fluid">
         <div className="container" style={{marginBottom: 60+'px'}}>
-          <form ref='form' action="/users" method="post">
+          <form ref='form' action="/movies" method="post">
             <div className="form-group">
               <label>Search Movies</label>
               <input type="text" ref="query" name="query" className="form-control" placeholder="Try a search like 'star wars'" required />
             </div>
-            <button type="submit" className="btn btn-primary" onClick={this.search}>Search</button>
+            <div className="form-group">
+              <button type="submit" className="btn btn-primary" onClick={this.search}>Search</button>
+            </div>
+            { !_.isEmpty(movies)
```

Comment:
this is a bit hard to read, consider extracting into two methods.

e.g.

`noResults () { return <p>No Results</p>; }`